config/purposes/scrna_core_cell_extraction.yml

```diff
@@ -72,10 +72,10 @@ LRC PBMC Bank:
     - name: 'Download PBMC Bank Tubes Content Report'
       id: pbmc_bank_tubes_content_report
       states: [passed, qc_complete]
-    - name: 'Download Cellaca Max 4 Count CSV'
-      id: hamilton_lrc_pbmc_bank_to_cellaca_max_4_count
-    - name: 'Download Cellaca ALL Count CSV'
-      id: hamilton_lrc_pbmc_bank_to_cellaca_all_count
+    - name: 'Download Cellaca FIRST Count CSV'
```

Comment:
I assume they proposed these names? Don't seem very descriptive but I guess they will have it written in their SOP what file they need when.

Reply:
Yes, they wanted Hamilton to Cellaca download buttons look different from other buttons on the page and shorter. They also wanted the words FIRST and SECOND in caps. The words refer to "replicates" as they described them.